app/controllers/api/v1x0/portfolio_items_controller.rb

```diff
@@ -36,6 +36,15 @@ def destroy
         head :no_content
       end
 
+      def copy
+        access_check('write', params.require(:portfolio_item_id))
```

Comment:
@lindgrenj6 trying to understand what this check does, I am thinking you need read access to the item you want to copy.
Say P1 is portfolio that is owned by Admin or some other owner (O1)
Owner 2 (O2) logs in so we have to first check the following
  1. Does O2 have write access to Portfolio, can he create a new portfolio (without any ids)
  2(a). Does O2 have read access to P1 so that It can copy it as P2 
           or
  2(b). Does O2 have write access to P1 so that it can copy it as P2

Reply:
Oh yeah that's definitely right - i'll change it to read access on the portfolio_item, write access on the portfolio.